#磁导信息消息体
float32 fLineV
float32 fAngleV
uint8 iNavType
int32 iRFID
uint8 iMLSEMotor
uint8 iDirection
